Objects/methodobject.c

```diff
@@ -203,7 +203,7 @@ static PyGetSetDef meth_getsets [] = {
 #define OFF(x) offsetof(PyCFunctionObject, x)
 
 static PyMemberDef meth_members[] = {
-    {"__module__",    T_OBJECT,     OFF(m_module), PY_WRITE_RESTRICTED},
+    {"__module__",    T_OBJECT,     OFF(m_module), 0},
```

Comment:
Maybe I'm not understanding the #define properly, but isn't PY_WRITE_RESTRICTED defined to have a value of 4? Why replace it with a 0? https://github.com/python/cpython/blob/007d7ff73f4e6e65cfafd512f9c23f7b7119b803/Include/structmember.h#L62

Reply:
The whole point of the issue is to stop using `PY_WRITE_RESTRICTED`.